components/02-elements/price/Price.vue

```diff
@@ -5,48 +5,94 @@
       {
         'price--special': special,
         'price-old': old
-      },
+      }
     ]"
   >
-    <slot />
+    <ins
+      v-if="special"
+      :aria-label="ariaLabelSpecial"
+    >
+      {{ computedValue }}
+    </ins>
+
+    <del
+      v-else-if="old"
+      :aria-label="ariaLabelOld"
+    >
+      {{ computedValue }}
+    </del>
+
+    <span v-else>
+      {{ computedValue }}
+    </span>
   </span>
 </template>
 
 <script>
-  export default {
-    props: {
-      special: {
-        type: Boolean,
-        default: false
-      },
-      old: {
-        type: Boolean,
-        default: false
+export default {
+  props: {
+    value: {
+      type: String,
+      required: true
+    },
+    currency: {
+      type: String,
+      default: null
+    },
+    prefix: {
+      type: String,
+      default: null
+    },
+    ariaLabelSpecial: {
+      type: String,
+      default: null
+    },
+    ariaLabelOld: {
+      type: String,
+      default: null
+    },
+    special: {
+      type: Boolean,
+      default: false
+    },
+    old: {
+      type: Boolean,
+      default: false
+    }
+  },
+  computed: {
+    computedValue() {
+      if(this.prefix) {
+        return this.prefix + this.value
+      }
+      if (this.currency) {
+        return this.value + ' ' + this.currency
       }
```

Comment:
Why currency at the end? What if we would like to have depending on the currency ($ or zl) at the end or at the front on different store views?

Reply:
We should get the formatted price from the outside (for example VSF), this is always tricky to decide how to format it.